src/plugins/sails.js

```diff
@@ -1,22 +1,22 @@
-import store from '../store/store.js';
-let _ = require('lodash');
+import store from "../store/store.js";
+let _ = require("lodash");
```

Comment:
Any reason for the double quotes? Most strings throughout the rest of the application are single quotes unless single quotes unless a literal ' is needed etc.

Reply:
No particular reason. Just what I've been using with default linter options.

Reply:
We should look into linter rules. I'll disable Prettier on my side for this project for now before it messes up with more code.